contracts/RewardManager.sol

```diff
@@ -127,7 +134,7 @@ contract RewardManager is Ownable, Versionable, Safe {
     emit RewardProgramCreated(rewardProgramID, admin);
   }
 
-  function removeRewardProgram(address rewardProgramID) external onlyOwner {
+  function removeRewardProgram(address rewardProgramID) external onlyGovernanceAdmin {
```

Comment:
This now means the owner cannot call the method, _only_ the governance admin.

Does it make sense to have a `onlyOwnerOrGovernanceAdmin` modifier for this purpose?

Reply:
I wud think to keep it simple. The owner is the deployer. The governance admin is a dao or an eoa under our control.  There is nothing stopping the the owner to configure the governance admin to point to itself. This edit wud just be purely for convenience -- don't think we need that.